src/client/client.tsx

```diff
@@ -22,9 +29,69 @@ export function fetchAllEvents(): Promise<Event[]> {
 export function fetchMember(nuid: string): Promise<Member | undefined> {
   return new Promise((resolve, reject) => {
     setTimeout(() => {
-      const member = (UserJSON as unknown as Member[]).find((m) => m.nuid === nuid);
+      const member = (UserJSON as unknown as Member[]).find(
+        (m) => m.nuid === nuid
+      );
       console.log("fetchMember", member);
       resolve(member);
     });
-  })
+  });
+}
+
+//sample function to fetch all attendance change requests
+export function findAttendanceChangeRequests(
```

Comment:
nit: Let's change all function keywords to ES6 arrow functions (less words make me happy lmao)

Reply:
export const fetchEvent = (id: Number): Promise<Event> => {
...
}